### PR TITLE
Refresh: Copy updates

### LINKED ISSUE #15649: Change pencil banner text and link in updated design.
## Description

Text should change to: `Donate to Mozilla by Dec 31 — `
CTA text should change to: `Donate ♥`
Link should change to: `https://foundation.mozilla.org/?form=eoy-pencil-aa`

image

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Final pass at copy

## Significant changes and points to review

- Check all text is translatable
- Wrap alt text for translation
- Copy change to pencil banner
- Copy change for title and description
- Add link attributes to some links

## Issue / Bugzilla link

Fix #15649

## Testing

Updates were made to both the home and about pages.

### CODE REVIEW

File: l10n/en/banners/m24-pencil-banner.ftl [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 # Variables:
 # $donate (url) - link to https://foundation.mozilla.org/en/?form=donate-today
-m24-pencil-banner-support-ethical = Support ethical AI and internet privacy — <strong><a { $donate }>donate today!</a></strong>
```
Comment (alexgibson) [inline l10n/en/banners/m24-pencil-banner.ftl:7]: **Issue:** might we want to fallback to this older string after Dec 31st?

File: bedrock/mozorg/templates/mozorg/home/includes/m24/ai-gallery.html [hunk 2]
```diff
@@ -55,7 +55,7 @@ <h2 class="m24-c-intro-title">{{ ftl('m24-home-join-the-movement') }}</h2>
         desc=ftl('m24-home-donate-your-voice-to-make'),
         link_url='https://commonvoice.mozilla.org/'+ utm_params,
         cta_text=ftl('m24-home-donate-your-voice'),
-        link_attributes='data-cta-text="Donate your voice"'
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/home/includes/m24/ai-gallery.html:58]: **Issue:** this gallery image is missing alt text still?

Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/ai-gallery.html:58]: That's intentional. It's a stock photo and doesn't supply any additional info.

File: bedrock/base/templates/includes/banners/pencil-banner.html [hunk 3]
```diff
@@ -4,12 +4,12 @@
...
 
-{% if not hide_pencil_banner and ftl_file_is_active('m24-pencil-banner') %}
+{% if not hide_pencil_banner and ftl_has_messages('m24-pencil-banner-donate-by'), %}
 {# Exclude banners from search result snippets using `data-nosnippet` (issue 8739) #}
 <aside class="m24-pencil-banner" aria-label="{{ ftl('ui-promo-label') }}" data-nosnippet="true">
   <div class="m24-pencil-banner-copy">
-    {% set donate_url = 'href="https://foundation.mozilla.org/?form=donate-today" data-cta-type="link" data-cta-text="donate today"' %}
-    <p>{{ ftl('m24-pencil-banner-support-ethical', donate=donate_url) }}</p>
+    {% set donate_url = 'href="https://foundation.mozilla.org/?form=eoy-pencil-aa" data-cta-type="link" data-cta-text="donate today"' %}
+    <p>{{ ftl('m24-pencil-banner-donate-by', donate=donate_url) }}</p>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/base/templates/includes/banners/pencil-banner.html:12]: ✅ I added a switch. I'm checking on the old string.

[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/base/templates/includes/banners/pencil-banner.html:12]: The old string should be removed.

File: bedrock/mozorg/templates/mozorg/home/includes/m24/ai-gallery.html [hunk 4]
```diff
@@ -178,15 +178,15 @@ <h2 class="m24-c-intro-title">{{ ftl('m24-home-join-the-movement') }}</h2>
...
+            "alt": ftl('m24-home-alt-ventures'),
           }
         ),
         tag=ftl('m24-home-tag-program'),
         title=ftl('m24-home-mozilla-ventures'),
         desc=ftl('m24-home-got-an-earlystage'),
         link_url='https://mozilla.vc/' + utm_params,
         cta_text=ftl('m24-home-read-more'),
-        link_attributes='data-cta-text="Read more (Ventures)"'
+        link_attributes='rel="external" target="_blank" data-link-text="Read more (Ventures)"'
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/ai-gallery.html:189]: ✅  It mostly doesn't matter as along as they all match. They should all be using `data-link-text` now.

File: l10n/en/banners/m24-pencil-banner.ftl (hunk 1 above)
Comment (stephaniehobson) [reply l10n/en/banners/m24-pencil-banner.ftl:7]: ✅  Official word is no. I added a switch like you suggested so we can disable the banner then.

**Comment by stephaniehobson** [comment]
Thank you for the careful alt text review 🙂 These two are missing it intentionally.

Updated and ready for re-review.

**Review by alexgibson — APPROVED** [review]
r+

**Pull request merged**